### ISSUE TITLE
Build warning from webpack

### ISSUE THREAD

**Issue by epatters** [nozzlegear/logspect#1]
When I use logspect (as a dependency of davenport) with webpack and TypeScript, I see the warning:

```
WARNING in ./node_modules/logspect/bin/index.js
(Emitted value instead of an instance of Error) Cannot find source file '../index.ts': Error: Can't resolve '../index.ts' in '[my package]/node_modules/logspect/bin'
 @ ./node_modules/davenport/bin/index.js 57:17-36
```

Nothing seems to be broken in the build but it'd be nice to get rid of the warning. Possibly related to commit https://github.com/nozzlegear/logspect/commit/41b2f192f518397aec96664772123e8c13960b62

**Comment by nozzlegear** [comment]
Thanks for letting me know about this. Should be an easy fix, I'll get it published today.

**Comment by nozzlegear** [comment]
Okay I *think* I've fixed this by adding the index.ts file to the list of published files. Should be available as version 1.3.1 on NPM, and I've also updated Davenport (2.8.1) to use this new version. Let me know if the problem persists.

**Comment by epatters** [comment]
Thanks. That fixed the warning from logspect. That's progress!

On one of my two projects using davenport (but for unknown reasons not the other), I see a similar warning from davenport itself:

```
WARNING in ./node_modules/davenport/bin/index.js
(Emitted value instead of an instance of Error) Cannot find source file '../index.ts': Error: Can't resolve '../index.ts' in '[my package]/node_modules/davenport/bin'
```

Can you please try making the same fix to davenport?

**Comment by nozzlegear** [comment]
D'oh, of course it needs to be done for Davenport too. Should be fixed in 2.8.2!

**Comment by epatters** [comment]
That did it. Thank you!
